### DISCUSSION TITLE
Add git annotation per line in github interface

### DISCUSSION THREAD

**Discussion by reahaas** [community/community#13696] [Projects and Issues]
When I use the Pycharm IDE, I can right-click and see the git annotation per line. It's can help to see who wrote it and when:
![image](https://user-images.githubusercontent.com/35425887/160391998-0964c98b-340a-40bd-9b3b-c875dbdc3f7a.png)

I would like to have the same behavior in the Github web interface:
![image](https://user-images.githubusercontent.com/35425887/160392282-99a9a667-c34e-47cd-8a28-06341778d8c3.png)

**Comment by daniel-portnoy** [comment]
Ya! Wow looks like something that will help all the Platform's users!
You really should implement this one!
